src/components/FilmDetails/FilmDetails.js

```diff
@@ -1,15 +1,23 @@
 /* eslint-disable max-len */
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { store } from '../../store/index';
 
 export class FilmDetails extends Component {
+  state = {
+    film: store.getState().films
+      .find(f => String(f.id) === this.props.match.params.id),
```

Comment:
What if id will change?

Reply:
everything will be fine because we have dynamic link if id changes link changes as well